pkg/metrics/metrics.go

```diff
@@ -0,0 +1,158 @@
+//
+// Copyright (c) 2018 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
+package metrics
+
+import (
+	"github.com/heketi/heketi/apps"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"net/http"
+)
```

Comment:
The standard in the code is std library modules up top. Sorry if I wasn't clear about that before.